Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ContestsController.cs

```diff
@@ -559,23 +559,21 @@ public ActionResult CalculateContestLoad(int categoryId, int currentContestId)
                 {
                     model.ExamLengthInHours = (contest.EndTime.Value - contest.StartTime.Value).Hours;
                     model.ExpectedExamProblemsCount = contest.ProblemGroups.Count();
-                    model.ExpectedStudentsCount = contest.Participants.Count();
                     model.ContestName = contest.Name;
                     model.CurrentContestId = contest.Id;
-
-                    model.AverageProblemRunTimeInSeconds = 
+                    model.AverageProblemRunTimeInSeconds =
                         this.contestsBusiness.GetContestSubmissionsAverageRunTimeSeconds(contest);
+
+                    // Currently is setted to 0 because it is not fetched from the SULS
+                    model.ExpectedStudentsCount = 0;
                 }
                 else
                 {
                     model.ContestsDropdownData.Add(new PreviousContestLoadData(contest));
                 }
             }
 
-            if (model.ContestsDropdownData.Any())
-            {
-                model.PreviousContestId = model.ContestsDropdownData.First().Id;
-            }
+            model.ActualWorkers = int.Parse(this.Data.Settings.All().First(x => x.Name == GlobalConstants.RemoteWorkers).Value);
```

Comment:
This will throw exception if the setting is deleted. I would handle this case with default value.

Reply:
Done